gripper_controllers/include/gripper_controllers/gripper_action_controller_impl.hpp

```diff
@@ -164,6 +165,7 @@ void GripperActionController<HardwareInterface>::check_for_success(
     pre_alloc_result_->position = current_position;
     pre_alloc_result_->reached_goal = true;
     pre_alloc_result_->stalled = false;
+    RCLCPP_INFO(get_node()->get_logger(), "Successfully moved to goal.");
```

Comment:
I would be happy to remove these print statements or change them to `DEBUG` if anyone prefers.

I added them because I had MTC stages that were failing when the gripper did not make it to the goal and nothing was output to the terminal saying what failed.

Reply:
Yes please

Reply:
Looks good otherwise